### PR TITLE
Import secrets are received without any terminal history

### PR DESCRIPTION
**Pull request by AbineshECAD**
**Existing behaviour:**
Secret key is part of the import cli command and available in cli history.
**Behaviour with this fix**
Secret key is not part of import cli. It is received fro user like a password. Not visible in cli history.

### CODE REVIEW

**Comment by AbineshECAD** [comment]
Test fro empty secret input:
```
% ./signatory-cli import --vault kms --base-dir ./ -c ./s.yaml
INFO[0000] Initializing vault                            vault=cloudkms vault_name=kms
Enter secret key: 
Error: enter a valid secret key

```
Positive import for encrypted key:
```
% ./signatory-cli import --vault kms --base-dir ./ -c ./s.yaml
INFO[0000] Initializing vault                            vault=cloudkms vault_name=kms
Enter secret key: 
Enter Password: 
INFO[0004] Requesting import operation                   pkh=tz3be5v4ZWL3zQYUZoLWJQy8P3H6RJryVVXn vault=CloudKMS vault_name=projects/signatory-testing/locations/europe-north1/keyRings/signatory-abi
Error: (CloudKMS/projects/signatory-testing/locations/europe-north1/keyRings/signatory-abi) CreateCryptoKey: rpc error: code = PermissionDenied desc = Permission 'cloudkms.cryptoKeys.create' denied on resource 'projects/signatory-testing/locations/europe-north1/keyRings/signatory-abi/cryptoKeys/signatory-imported-2IvBAi9qqwUy5X6f6ZqEs6sE2of' (or it may not exist).

```

File: cmd/commands/import.go [hunk 1]
```diff
@@ -17,9 +17,9 @@ func NewImportCommand(c *Context) *cobra.Command {
 	)
 
 	importCmd := &cobra.Command{
-		Use:   "import <secret-key>",
+		Use:   "import <flags>",
```
Comment (danielelisi) [inline cmd/commands/import.go:20]: I think you should enclose `flags` around `[]` as squared brakets are usually used for optional parameters in CLI utilities

File: cmd/commands/import.go [hunk 2]
```diff
@@ -31,6 +31,7 @@ func NewImportCommand(c *Context) *cobra.Command {
 				passCB = func() ([]byte, error) { return []byte(password), nil }
 			} else {
 				passCB = func() ([]byte, error) {
+					fmt.Println()
 					fmt.Print("Enter Password: ")
```
Comment (danielelisi) [inline cmd/commands/import.go:35]: Can this be refactored into one line ?
```
fmt.Println("Enter Password: ")
```

File: cmd/commands/import.go [hunk 3]
```diff
@@ -41,11 +42,19 @@ func NewImportCommand(c *Context) *cobra.Command {
 				options[k] = v
 			}
 
-			for _, key := range args {
-				_, err := c.signatory.Import(c.Context, vaultName, key, passCB, options)
-				if err != nil {
-					return err
-				}
+			fmt.Print("Enter secret key: ")
```
Comment (danielelisi) [inline cmd/commands/import.go:45]: What's the difference between `enter secret key` and `Enter Password` ?

File: cmd/commands/import.go (hunk 1 above)
Comment (AbineshECAD) [reply cmd/commands/import.go:20]: Done

File: cmd/commands/import.go (hunk 2 above)
Comment (AbineshECAD) [reply cmd/commands/import.go:35]: Password is applicable only for encrypted keys. So this prompt appears only for encrypted keys, not required for un-encrypted keys.

File: cmd/commands/import.go (hunk 3 above)
Comment (AbineshECAD) [reply cmd/commands/import.go:45]: Explanation: https://github.com/ecadlabs/signatory/pull/267#discussion_r1080594783

Will it be apparent if the `Enter Password:` prompt is modified to `This is encrypted, enter the password:`?

Comment (danielelisi) [reply cmd/commands/import.go:45]: Oh I see! Yeah good idea, let's make it more explicit!

Comment (AbineshECAD) [reply cmd/commands/import.go:45]: Done

**Pull request merged**